kubernetes/linux/setup.sh

```diff
@@ -46,6 +46,11 @@ mv /usr/lib/ruby/gems/3.1.0/specifications/default/uri-0.11.0.gemspec /usr/lib/r
 mv /usr/lib/ruby/gems/3.1.0/specifications/default/stringio-3.0.1.gemspec /usr/lib/ruby/gems/3.1.0/specifications/default/..
 mv /usr/lib/ruby/gems/3.1.0/specifications/default/rexml-3.2.5.gemspec /usr/lib/ruby/gems/3.1.0/specifications/default/..
 mv /usr/lib/ruby/gems/3.1.0/specifications/webrick-1.8.1.gemspec /usr/lib/ruby/gems/3.1.0/specifications/..
+gem install cgi -v "0.3.7" --no-document
+gem uninstall cgi --version 0.3.5
+rm -f /usr/lib/ruby/gems/3.1.0/specifications/default/cgi-0.3.5.gemspec
```

Comment:
after fixing, do we have any other CVEs other than telegraf?

Reply:
nope: https://github-private.visualstudio.com/microsoft/_build/results?buildId=98197&view=logs&j=c275f7fe-e659-5231-3649-94d1ac43e3f4&t=447eaa17-544e-5284-11e4-7ff1233d0cf2